packages/@coorpacademy-components/src/atom/button-link-icon-only/index.js

```diff
@@ -0,0 +1,84 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {getOr} from 'lodash/fp';
+import classnames from 'classnames';
+import {ICONS} from '../../util/button-icons';
+import Link from '../link';
+import style from './style.css';
+
+const getButtonContent = icon => {
+  const Icon = getOr(null, icon, ICONS);
+
+  if (!Icon) {
+    return <div className={style.buttonContent} />;
+  }
+
+  return (
+    <div className={style.buttonContent}>
+      <Icon className={style.icon} />
+    </div>
+  );
+};
+
+const ButtonLinkIconOnly = props => {
+  const {
+    size = 'default',
+    disabled,
+    icon,
+    'data-name': dataName,
+    'aria-label': ariaLabel,
+    link,
+    onClick
+  } = props;
+  const contentView = getButtonContent(icon);
+  const styleButton = classnames(
+    style.button,
+    size === 'default' ? style.default : style.small,
```

Comment:
Soit tu supprimes cette ligne 
```suggestion
    size === 'default' ? style.default : style.small,
```
soit tu supprimes les `composes` dans le css de default et small , car button est appliqué deux fois